management_api.go

```diff
@@ -296,6 +296,33 @@ func (e *Enforcer) RemoveNamedGroupingPolicies(ptype string, rules [][]string) (
 	return e.removePolicies("g", ptype, rules)
 }
 
+func (e *Enforcer) UpdateGroupingPolicy(oldName string, newName string) (bool, error) {
+	return e.UpdateNamedGroupingPolicy("g", oldName, newName)
+}
+
+func (e *Enforcer) UpdateNamedGroupingPolicy(ptype string, oldName string, newName string) (bool, error) {
+	roles, err := e.GetRolesForUser(oldName)
+	if err != nil {
+		return false, err
+	}
+	names, err := e.GetUsersForRole(oldName)
+	if err != nil {
+		return false, err
+	}
+	for _, role := range roles {
+		if flag, err := e.updatePolicy("g", ptype, []string{oldName, role}, []string{newName, role}); err != nil || !flag {
```

Comment:
it doesn't seems correct to me because here we are assuming: `g` only have two parameters but it's not always this case

Reply:
Do you mean we should add UpdateRoleInDomain? 
Or is it like UpdatePolicy?
```
func (e *Enforcer) UpdateNamedPolicy(ptype string, p1 []string, p2 []string) (bool, error) {
	return e.updatePolicy("p", ptype, p1, p2)
}
```

Reply:
I think the new logic could make sure this problem.

Reply:
Any update?